app/javascript/components/mappers/domainToApi/application.js

```diff
@@ -24,6 +24,7 @@ export const applicationFieldMapper = {
   has_developmental_disability: 'hasDevelopmentalDisability',
   answered_community_screening: 'answeredCommunityScreening',
   annual_income: 'annualIncome',
+  confirmed_annual_income: 'confirmedHouseholdAnnualIncome',
```

Comment:
I'm also going to update this to be confirmed_household_annual_income. It's long but income fields are confusing enough that I think we need to be very consistent.